kano_updater/ui/install_window.py

```diff
@@ -54,6 +62,29 @@ def __init__(self):
 
         self._start_install()
 
+    def _launch_game(self, window=None, event=None):
+        try:
+            if event and event.get_keycode()[1] == 44:  # [J] key
+                if self._is_game_first_launch:
+                    self._is_game_first_launch = False
+                    self._first_game_launch()
+                else:
+                    self._relaunch_game()
+        except:
+            pass
+
+    def _first_game_launch(self):
+        self.set_keep_below(True)
+        os.system('{} &'.format(FLAPPY_PATH))
+        os.system('kill -9 `pgrep -f kano-feedback`')      # kill kano-feedback-widget
```

Comment:
It might be slightly more efficient to use `pkill -f -9`. That wouldn't launch two processes.